examples/templates/vue/README.md

```diff
@@ -0,0 +1,61 @@
+# {{ YOUR TITLE GOES HERE }}
+
+## Description
+
+{{ YOUR DESCRIPTION GOES HERE }}
+
+## Live preview
+
+You can preview this template on CodeSandbox  under the URL: https://githubbox.com/handsontable/code-snippets/tree/master/examples/templates/vue
+
+## How to run this project
+
+### Installation
+
+Call `npm install` to install all necessary dependencies. This command will first remove `node_modules` folder and install testing tools (`jasmine`, `jasmine-console-reporter`, `puppeteer` and `http-server`) from the root directory.
```

Comment:
Do not mention the deps, we will have to update readme each time we decide to change them. Dev deps should be internal and free to refactor anytime. Devs know what `npm install` does with `node_modules`

```suggestion
Call `npm install` to install all necessary dependencies.
```

Applies to all readme.md's

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/4cedf4a98e61e6ca3bd794aacadbac53b0f563fa